jobs/doppler/templates/doppler_ctl.erb

```diff
@@ -31,8 +31,12 @@ case $1 in
 
     <% p("doppler.debug") == true ? debug_string = "--debug " : debug_string = "" %>
 
+    <% if p('doppler.log_to_syslog') %>
+    tee_output_to_sys_log "$LOG_DIR" doppler stdout stderr
```

Comment:
We are already T-ing the logs to $LOGDIR and `logger` on line 20 above. Setting `doppler.log_to_syslog` to `true`seems redundant.